packages/analytics/client/src/analytics_client/context_service.ts

```diff
@@ -96,8 +95,8 @@ export class ContextService {
     );
   }
 
-  private removeEmptyValues(context?: Partial<EventContext>) {
-    if (!context) {
+  private removeEmptyValues(context: unknown) {
+    if (!isObject(context)) {
```

Comment:
Do we need a test for this new behaviour?

Reply:
It is tested (it tests if undefined is handled correctly). This was about fixing the types (and that forced me to add the type guard)